piu/cli.py

```diff
@@ -177,24 +180,58 @@ def cli(ctx, config_file):
 
 
 @cli.command('request-access')
-@click.argument('host', metavar='[USER]@HOST')
-@click.argument('reason')
-@click.argument('reason_cont', nargs=-1, metavar='[..]')
+@click.argument('host', metavar='[USER]@HOST', required=False)
+@click.argument('reason', required=False)
+@click.argument('reason_cont', nargs=-1, metavar='[..]', required=False)
 @click.option('-U', '--user', help='Username to use for OAuth2 authentication', envvar='PIU_USER', metavar='NAME')
 @click.option('-p', '--password', help='Password to use for OAuth2 authentication',
               envvar='PIU_PASSWORD', metavar='PWD')
 @click.option('-E', '--even-url', help='Even SSH Access Granting Service URL', envvar='EVEN_URL', metavar='URI')
 @click.option('-O', '--odd-host', help='Odd SSH bastion hostname', envvar='ODD_HOST', metavar='HOSTNAME')
 @click.option('-t', '--lifetime', help='Lifetime of the SSH access request in minutes (default: 60)',
               type=click.IntRange(1, 525600, clamp=True))
+@click.option('--interactive', help='Offers assistance', envvar='PIU_INTERACTIVE', is_flag=True, default=False)
 @click.option('--insecure', help='Do not verify SSL certificate', is_flag=True, default=False)
-@click.option('--clip', is_flag=True, help='Copy SSH command into clipboard', default=False)
-@click.option('--connect', is_flag=True, help='Directly connect to the host', default=False)
+@click.option('--clip', help='Copy SSH command into clipboard', is_flag=True, default=False)
+@click.option('--connect', help='Directly connect to the host', envvar='PIU_CONNECT', is_flag=True, default=False)
 @click.pass_obj
-def request_access(obj, host, user, password, even_url, odd_host, reason, reason_cont, insecure, lifetime,
-                   clip, connect):
+def request_access(obj, host, reason, reason_cont, user, password, even_url, odd_host, lifetime, interactive,
+                   insecure, clip, connect):
     '''Request SSH access to a single host'''
 
+    if interactive:
+        ec2 = boto3.resource('ec2')
```

Comment:
assumes the current region `AWS_DEFAULT_REGION` or `~/.aws/config` is correct --- that differs from other STUPS' tools